### PR TITLE
update DynamicResourcePool.java

### PR DESCRIPTION
**Pull request by slwle123**
remove import com.google.common.base.Throwables;

replace throw throwables.propagete(e ) with throw new RuntimeException(e);

### CODE REVIEW

**Comment by aaudiber** [comment]
Hey @slwle123, please address the comments from `alluxio-bot` and sign the CLA (link [here](http://cla.alluxio.org/)). Thanks!

File: core/common/src/main/java/alluxio/resource/DynamicResourcePool.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import alluxio.clock.SystemClock;
 
 import com.google.common.base.Preconditions;
-import com.google.common.base.Throwables;
+//import com.google.common.base.Throwables;
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/resource/DynamicResourcePool.java]: You should delete the deadcode

File: core/common/src/main/java/alluxio/resource/DynamicResourcePool.java [hunk 2]
```diff
@@ -290,7 +290,8 @@ public T acquire() throws IOException, InterruptedException {
       return acquire(100  /* no timeout */, TimeUnit.DAYS);
     } catch (TimeoutException e) {
       // Never should timeout in acquire().
-      throw Throwables.propagate(e);
+      //throw Throwables.propagate(e);
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/resource/DynamicResourcePool.java]: ditto

**Comment by yupeng9** [comment]
@slwle123 any updates?

**Comment by yupeng9** [comment]
@slwle123 feel free to reopen this PR after you address the comments.

**Pull request closed without merging**